include/MeshManager.hpp

```diff
@@ -125,6 +134,12 @@ namespace blitzdg {
        */
       index_type get_ElementType() const;
 
+      /**
+       * Returns table of boundary condition tags corresponding
+       * to element faces. (length = NumElements * NumFaces).
+       */
+      index_vector_type& get_BCType() const;
```

Comment:
Return by const reference to `index_vector_type`.

Reply:
Fixed in 1147657cbdbc.